src/umpire/resource/FileMemoryResource.cpp

```diff
@@ -0,0 +1,104 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#ifndef UMPIRE_FileMemoryResource_INL
```

Comment:
Remove this line.  It is only used in the header file to protect against multiple inclusion and doesn't belong in the .cpp file.